docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,190 @@
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node2.yml
+
+
+The :file:`docker-compose.yml` file describes your distributed application (:file:`docker-compose.yml` was formerly known as :file:`fig.yml`):
+
+.. literalinclude:: docker-compose.yml
+   :language: yaml
+
+The :file:`deployment-node1.yml` file describes which containers to deploy, and where:
+
+.. literalinclude:: deployment-node1.yml
+   :language: yaml
+
+Now you can use the Flocker CLI to deploy both your web application and server onto one of the virtual machines you have just created:
+
+.. prompt:: bash you@laptop:~$
+
+   flocker-deploy 172.16.255.250 deployment-node1.yml docker-compose.yml
+
+* Visit http://172.16.255.250/.
+  You will see the visit count displayed.
+* Visit http://172.16.255.251/.
+  You will see that the count persists because Flocker routes the traffic from either node named in the deployment file to the one that has the application.
+
+Migrating a Container to the Second Host
+========================================
+
+The diagram below illustrates your current server-side Flocker setup:
+
+.. image:: images/flocker-tutorial-initial-setup.svg
+   :width: 60 %
+   :alt: In the server-side Flocker setup there are two servers, one of which has two Docker containers running; one container is a running a web application, the other has a Redis database with a volume.
+   :align: center
+
+To move the container with the Redis server along with its data volume, use the :file:`deployment-node2.yml` file:
+
+.. literalinclude:: deployment-node2.yml
+   :language: yaml
+
+Now you can use the Flocker CLI to migrate one of the containers to the second host:
+
+.. prompt:: bash you@laptop:~$
+
+   flocker-deploy 172.16.255.250 deployment-node2.yml docker-compose.yml
+
+The container on the Redis server and its volume have now both been moved to the second host, and Flocker has maintained its link to the web application on the first host:
+
+* Visit http://172.16.255.250/.
+  You will see the visit count is still persisted.
+* Visit http://172.16.255.251/.
+  You will see that the count still persists, even though the container with the volume has moved between hosts.
+
+Result
+======
+
+You have now set up your first Flocker cluster and moved a Docker container with its volume, while persisting its link to a web app on another server.
+
+The following diagram illustrates how your server-side Flocker setup looks now:
+
+.. image:: images/flocker-tutorial-final-setup.svg
+   :width: 60 %
+   :alt: The web application is still running within a container on the first server, while the Redis server with a volume is now running on the second server.
+   :align: center
+
+The next Flocker tutorial is :ref:`Deploying and Migrating MongoDB <tutmongo>`, which will teach you how to use Flocker's container, network, and volume orchestration functionality, based around the setup of a MongoDB service.
```

Comment:
> based around the setup of a MongoDB service.

:question: MongoDB is an implementation detail of that tutorial. Perhaps better is ", using MongoDB as an example".